examples/icd_code_prediction/README.md

```diff
@@ -0,0 +1,46 @@
+## A ICD Code Prediction Demo
+
+This demo shows how we can build pipeline to predict ICD-10 codes for medical notes.
+
+The [International Classification of Diseases (ICD)](https://www.cdc.gov/nchs/icd/index.htm) is the classification used to code and classify mortality data from death certificates. We treat the ICD code prediction as a sequence classification task. That is, given an medical note, the model output its ICD code.
+
+A key component of this demo is `ICDCodingProcessor`, which is implemented based on [AkshatSurolia's pre-trained model](https://huggingface.co/AkshatSurolia/ICD-10-Code-Prediction).
```

Comment:
The model is actually configurable via [this parameter](https://github.com/KiaLAN/ForteHealth/blob/60bb3919d4f96f246f328f5f8a86bc771f6bba04/fortex/health/processors/icd_coding_processor.py#L106). So this description is not exactly accurate. In addition, it might be helpful to show how to configure this.